README.md

```diff
@@ -58,24 +68,19 @@ Not literally “protect from”. The plugin’s purpose is to _detect_ any “h
 * updated [plugin authors](https://gist.github.com/glueckpress/f058c0ab973d45a72720)
 
 ### 1.3.8 ###
-* Deutsch: Erkennung der [MailPoet-Sicherheitslücke](http://blog.sucuri.net/2014/07/mailpoet-vulnerability-exploited-in-the-wild-breaking-thousands-of-wordpress-sites.html)
-* English: Detection and warning for the [MailPoet Vulnerability](http://blog.sucuri.net/2014/07/mailpoet-vulnerability-exploited-in-the-wild-breaking-thousands-of-wordpress-sites.html)
+* Detection and warning for the [MailPoet Vulnerability](http://blog.sucuri.net/2014/07/mailpoet-vulnerability-exploited-in-the-wild-breaking-thousands-of-wordpress-sites.html)
 
 ### 1.3.7 ###
-* Deutsch: Aktualisierung auf Safe Browsing Lookup API 3.1
-* English: Update the Google Safe Browsing Lookup API to v3.1
+* Update the Google Safe Browsing Lookup API to v3.1
 
 ### 1.3.6 ###
-* Deutsch: Code-Revision und Datenvalidierung
-* English: Code revision and data validation
+* Code revision and data validation
 
 ### 1.3.5 ###
-* Deutsch: Optimierungen für WordPress 3.8
-* English: Optimizations for WordPress 3.8
+* Optimizations for WordPress 3.8
 
 ### 1.3.4 ###
-* Deutsch: Benachrichtigung per E-Mail, sobald [Google Safe Browsing](http://en.wikipedia.org/wiki/Google_Safe_Browsing) Malware im Blog erkennt. [Mehr auf Google+](https://plus.google.com/110569673423509816572/posts/H72FFwvna1i)
-* English: [Google Safe Browsing](http://en.wikipedia.org/wiki/Google_Safe_Browsing) for malware and phishing monitoring.
+* [Google Safe Browsing](http://en.wikipedia.org/wiki/Google_Safe_Browsing) for malware and phishing monitoring with e-mail notification. [Additional informations on Google+](https://plus.google.com/110569673423509816572/posts/H72FFwvna1i) (only german)
```

Comment:
Information is always singular in English. Languages like German are always capitalized.